packages/editor/src/components/rich-text/index.native.js

```diff
@@ -40,11 +40,12 @@ const FORMATTING_CONTROLS = [
 		title: __( 'Italic' ),
 		format: 'italic',
 	},
-	{
-		icon: 'admin-links',
-		title: __( 'Link' ),
-		format: 'link',
-	},
+	// TODO: get this back after alpha
```

Comment:
I think we can remove the code, we use a VCS after all :)

Reply:
heh ofc, the thing is given this is a last minute thing and we'll need it back to continue work after alpha, it makes sense to keep it calling our attention "on the foreground" :)